examples/src/Utils/Animated.elm

```diff
@@ -93,3 +96,28 @@ animatedTypedSvg node animation attributes children =
                 (TypedSvg.style [] [ TypedSvg.text stylesheet ] :: children)
         )
         animation
+
+
+
+-- elm-css
+
+
+animatedStyledNode :
+    (List (Htmls.Attribute msg) -> List (Htmls.Html msg) -> Htmls.Html msg)
+    -> Animation
+    -> List (Htmls.Attribute msg)
+    -> List (Htmls.Html msg)
+    -> Htmls.Html msg
+animatedStyledNode nodeToAnimate animation attributes children =
+    Animated.custom
+        (\className stylesheet ->
+            nodeToAnimate
+                (Attr.class className :: attributes)
```

Comment:
The type signatures are a bit hard to guess but the way I've converted these to library helpers is to replace any UI specific functions like `Attr.class` or `Htmls.text` with `options.class` and `options.text`

So you'd end up with something like:

```elm
animatedStyledNode options nodeToAnimate animation attributes children =
    custom
        (\className stylesheet ->
            nodeToAnimate
                (options.class className :: attributes)
                (options.node "style" [] [ options.text stylesheet ] :: children)
        )
        animation
```

I arrived at this:

```elm
type alias ElmCssOptions attr node =
    { class : ClassName -> attr
    , node : String -> List attr -> List node -> node
    , text : Stylesheet -> node
    }


styled : ElmCssOptions attr node -> (List attr -> List node -> node) -> Animation -> List attr -> List node -> node
styled options nodeToAnimate animation attributes children =
    custom
        (\className stylesheet ->
            nodeToAnimate
                (options.class className :: attributes)
                (options.node "style" [] [ options.text stylesheet ] :: children)
        )
        animation

```